lib/api-v1.js

```diff
@@ -506,7 +506,7 @@ api.declare({
   let input = req.body;
   let token = req.params.token;
 
-  await req.authorize({workerType: workerType});
+  await req.authorize({workerType: input.workerType});
```

Comment:
Nice fix! Did the test crash before, or fail/pass silently? Too bad JS isn't stricter to automatically spot things like this.